info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -283,20 +288,33 @@ private void storeElectionContestCandidateInDatabase(JsonObject candidate,
         Entity.newBuilder(candidateKey)
             .set("name", name)
             .set("partyAffiliation", party + " Party")
+            .set("email", email)
+            .set("phone number",phoneNumber)
+            .set("photoURL", photo)
+            .set("website", website)
             .build();
     datastore.put(candidateEntity);
     candidateIds.add(StringValue.newBuilder(Long.toString(candidateId)).build());
     candidateIncumbency.add(BooleanValue.newBuilder(false).build());
 
     compileAndStoreCandidateNewsArticlesInDatabase(name, new Long(candidateId).toString());
   }
+  private String getField(JsonObject candidate, String field) {
+    if (candidate.has(field)) {
+      return candidate.get(field).getAsString();
+    } else {
+      return "not available";
```

Comment:
Return an empty string string. If you're using the result of this method later on, a "not available" result would falsely pass the .isEmpty() check